<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1326</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1321</width>
      <height>571</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mongolian Baiti</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Gestion Entreprise</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1321</width>
        <height>541</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>AJOUTER</string>
       </attribute>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>40</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Nom</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>100</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Secteur</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>160</y>
          <width>63</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Adresse</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>230</y>
          <width>63</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Email</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>22</x>
          <y>300</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Telephone</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="nom_e">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>30</y>
          <width>113</width>
          <height>26</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="secteur_e">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>90</y>
          <width>113</width>
          <height>26</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="adresse_e">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>150</y>
          <width>113</width>
          <height>26</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="email_e">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>220</y>
          <width>113</width>
          <height>26</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="tel_e">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>290</y>
          <width>113</width>
          <height>26</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="ajouter_e">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>210</y>
          <width>121</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>AJOUTER</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>AFFICHER</string>
       </attribute>
       <widget class="QTableView" name="affiche_e">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1331</width>
          <height>521</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="id_sup">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>370</y>
          <width>113</width>
          <height>26</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>410</y>
          <width>101</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>SUPPRIMER</string>
        </property>
       </widget>
       <widget class="QComboBox" name="criter">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>370</y>
          <width>76</width>
          <height>26</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>id</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>nom</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>secteur</string>
         </property>
        </item>
       </widget>
       <widget class="QPushButton" name="tri_e">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>410</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>TRI</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="rech_e">
        <property name="geometry">
         <rect>
          <x>570</x>
          <y>370</y>
          <width>113</width>
          <height>26</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="recherche_e">
        <property name="geometry">
         <rect>
          <x>570</x>
          <y>410</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>RECHERCHE</string>
        </property>
       </widget>
       <widget class="QPushButton" name="PDF_e">
        <property name="geometry">
         <rect>
          <x>1090</x>
          <y>370</y>
          <width>101</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string> PDF </string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>MODIFIER</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1271</width>
          <height>471</height>
         </rect>
        </property>
        <property name="title">
         <string>NOUVELLES VALEURS</string>
        </property>
        <widget class="QCommandLinkButton" name="commandLinkButton">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>140</y>
           <width>61</width>
           <height>47</height>
          </rect>
         </property>
         <property name="text">
          <string>ID</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="nv_ide">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>110</y>
           <width>113</width>
           <height>26</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>310</x>
           <y>50</y>
           <width>63</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Nom</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>310</x>
           <y>120</y>
           <width>63</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Secteur</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>310</x>
           <y>200</y>
           <width>63</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Adresse</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_9">
         <property name="geometry">
          <rect>
           <x>310</x>
           <y>280</y>
           <width>63</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Email</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_10">
         <property name="geometry">
          <rect>
           <x>292</x>
           <y>360</y>
           <width>81</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Telephone</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="nv_nom">
         <property name="geometry">
          <rect>
           <x>390</x>
           <y>40</y>
           <width>151</width>
           <height>26</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="nv_secteur">
         <property name="geometry">
          <rect>
           <x>390</x>
           <y>110</y>
           <width>151</width>
           <height>26</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="nv_adresse">
         <property name="geometry">
          <rect>
           <x>390</x>
           <y>200</y>
           <width>151</width>
           <height>26</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="nv_email">
         <property name="geometry">
          <rect>
           <x>390</x>
           <y>270</y>
           <width>151</width>
           <height>26</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="nv_tel">
         <property name="geometry">
          <rect>
           <x>390</x>
           <y>350</y>
           <width>151</width>
           <height>26</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton">
         <property name="geometry">
          <rect>
           <x>770</x>
           <y>280</y>
           <width>121</width>
           <height>81</height>
          </rect>
         </property>
         <property name="text">
          <string>MODIFIER</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>ARCHIVE</string>
       </attribute>
       <widget class="QTableView" name="archive_e">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1311</width>
          <height>511</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>STATISTIQUE</string>
       </attribute>
       <widget class="QGraphicsView" name="stat_e">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1311</width>
          <height>511</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Gestion Offre</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1321</width>
        <height>541</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_8">
       <attribute name="title">
        <string>AJOUTER</string>
       </attribute>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>40</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Titre</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>100</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Description</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>12</x>
          <y>160</y>
          <width>101</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Competences</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>230</y>
          <width>63</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Status</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>2</x>
          <y>300</y>
          <width>101</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>ID entreprise</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="titre_o">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>30</y>
          <width>113</width>
          <height>26</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="com_o">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>150</y>
          <width>113</width>
          <height>26</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="id_eo">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>290</y>
          <width>113</width>
          <height>26</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="ajouter_e_2">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>210</y>
          <width>121</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>AJOUTER</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="des_o">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>70</y>
          <width>104</width>
          <height>75</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="status_o">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>230</y>
          <width>76</width>
          <height>26</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>active</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>inactive</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="label_21">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>370</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Date Limite</string>
        </property>
       </widget>
       <widget class="QDateEdit" name="date_limite_o">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>370</y>
          <width>110</width>
          <height>26</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_9">
       <attribute name="title">
        <string>AFFICHER</string>
       </attribute>
       <widget class="QTableView" name="affiche_o">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1331</width>
          <height>521</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="id_supo">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>370</y>
          <width>113</width>
          <height>26</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>410</y>
          <width>101</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>SUPPRIMER</string>
        </property>
       </widget>
       <widget class="QComboBox" name="criter_2">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>370</y>
          <width>76</width>
          <height>26</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>id</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>titre</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>date_limite</string>
         </property>
        </item>
       </widget>
       <widget class="QPushButton" name="tri_e_2">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>410</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>TRI</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="rech_o">
        <property name="geometry">
         <rect>
          <x>570</x>
          <y>370</y>
          <width>113</width>
          <height>26</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="recherche_e_2">
        <property name="geometry">
         <rect>
          <x>570</x>
          <y>410</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>RECHERCHE</string>
        </property>
       </widget>
       <widget class="QPushButton" name="PDF_e_2">
        <property name="geometry">
         <rect>
          <x>1090</x>
          <y>370</y>
          <width>101</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string> PDF </string>
        </property>
       </widget>
       <widget class="QLineEdit" name="pdf_o">
        <property name="geometry">
         <rect>
          <x>1090</x>
          <y>330</y>
          <width>113</width>
          <height>26</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_5">
        <property name="geometry">
         <rect>
          <x>860</x>
          <y>370</y>
          <width>111</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>LinkedIn</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_10">
       <attribute name="title">
        <string>MODIFIER</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1271</width>
          <height>471</height>
         </rect>
        </property>
        <property name="title">
         <string>NOUVELLES VALEURS</string>
        </property>
        <widget class="QCommandLinkButton" name="commandLinkButton_2">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>140</y>
           <width>61</width>
           <height>47</height>
          </rect>
         </property>
         <property name="text">
          <string>ID</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="nv_ido">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>110</y>
           <width>113</width>
           <height>26</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_16">
         <property name="geometry">
          <rect>
           <x>310</x>
           <y>50</y>
           <width>63</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Titre</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_17">
         <property name="geometry">
          <rect>
           <x>282</x>
           <y>120</y>
           <width>91</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Description</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_18">
         <property name="geometry">
          <rect>
           <x>282</x>
           <y>200</y>
           <width>91</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>competences</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_19">
         <property name="geometry">
          <rect>
           <x>310</x>
           <y>280</y>
           <width>63</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Status</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_20">
         <property name="geometry">
          <rect>
           <x>262</x>
           <y>360</y>
           <width>111</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>ID_entreprise</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="nv_titre">
         <property name="geometry">
          <rect>
           <x>390</x>
           <y>40</y>
           <width>151</width>
           <height>26</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="nv_com">
         <property name="geometry">
          <rect>
           <x>390</x>
           <y>200</y>
           <width>151</width>
           <height>26</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="nv_ideo">
         <property name="geometry">
          <rect>
           <x>390</x>
           <y>350</y>
           <width>151</width>
           <height>26</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_4">
         <property name="geometry">
          <rect>
           <x>770</x>
           <y>280</y>
           <width>121</width>
           <height>81</height>
          </rect>
         </property>
         <property name="text">
          <string>MODIFIER</string>
         </property>
        </widget>
        <widget class="QTextEdit" name="nv_des">
         <property name="geometry">
          <rect>
           <x>390</x>
           <y>110</y>
           <width>104</width>
           <height>75</height>
          </rect>
         </property>
        </widget>
        <widget class="QComboBox" name="nv_st">
         <property name="geometry">
          <rect>
           <x>390</x>
           <y>280</y>
           <width>76</width>
           <height>26</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>active</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>inactive</string>
          </property>
         </item>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_12">
       <attribute name="title">
        <string>STATISTIQUE</string>
       </attribute>
       <widget class="QGraphicsView" name="stat_o">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1311</width>
          <height>511</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_11">
       <attribute name="title">
        <string>HISTORIQUE</string>
       </attribute>
       <widget class="QTableView" name="his">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1321</width>
          <height>511</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
